gitops/bootstrap/control-plane/addons/oss/addons-kargo-appset.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
New eyes 👀 
Can we add more context on why we are adding this file for my sanity ? 
To me it looks like we are trying to install KArgo. But is that it?

Reply:
Yes